From bab129f11a61f5b12329778d0324879ded6744ba Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Sat, 8 Aug 2026 08:08:18 +0100
Subject: [PATCH] common: update submodule

  Richard W.M. Jones (2):
        mlcustomize/firstboot.ml: Print the version of Windows
        mlcustomize/firstboot.ml: Use rem for comments (instead of '::')

(cherry picked from commit da0c19d25df40e31d6bf7c8f349478ddf616e02b)
---
 common | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Submodule common 9c02b103..877c8ca0:
diff --git a/common/mlcustomize/firstboot.ml b/common/mlcustomize/firstboot.ml
index fc3b7a69..1e019ecf 100644
--- a/common/mlcustomize/firstboot.ml
+++ b/common/mlcustomize/firstboot.ml
@@ -299,12 +299,13 @@ exit /b
 :main
 echo starting firstboot service
 echo effective user: %%USERNAME%% domain: %%USERDOMAIN%%
+ver
 
 if not exist "%%scripts_done%%" (
   mkdir "%%scripts_done%%"
 )
 
-:: Pick the next script to run.
+rem Pick the next script to run.
 for %%%%f in ("%%scripts%%"\*.bat) do (
   echo running "%%%%f"
   pushd "%%scripts%%"
@@ -320,23 +321,23 @@ for %%%%f in ("%%scripts%%"\*.bat) do (
     echo Script failed, will retry on next boot
   )
 
-  :: Reboot the computer only if exit code indicates.
-  :: Exit code 250 means do not reboot, otherwise reboot
-  :: Reboot is necessary to free any locked files which
-  :: may prevent later scripts from running.
+  rem Reboot the computer only if exit code indicates.
+  rem Exit code 250 means do not reboot, otherwise reboot.
+  rem Reboot is necessary to free any locked files which
+  rem may prevent later scripts from running.
   if !elvl! NEQ 250 (
     shutdown /r /t 0 /y
 
-    :: Exit the script (in case shutdown returns before rebooting).
-    :: On next boot, the whole firstboot service will be called again.
-
+    rem Exit the script (in case shutdown returns before
+    rem rebooting).  On next boot, the whole firstboot
+    rem service will be called again.
     exit /b
   )
 )
 
-:: Fallthrough here if there are no scripts.
+rem Fallthrough here if there are no scripts.
 
-:: Touch a sentinel file to say we have finished.
+rem Touch a sentinel file to say we have finished.
 type nul > "%%firstboot%%\complete"
 
 echo uninstalling firstboot service
